### PR TITLE
st1 [2034][ADD] stock_picking_partner_info

### PR DESCRIPTION
**Pull request by tack-yabeQuartile**
[#2034](https://www.quartile.co/web#id=2034&model=project.task&view_type=form&menu_id=)

* Display address and phone at delivery form.

This PR supersedes [rpl-private 223.](https://github.com/qrtl/rpl-private/pull/223)

### CODE REVIEW

**Comment by tack-yabeQuartile** [comment]
Commands below was used to make this PR.
```
cd rpl-custom
git checkout st1
git pull --ff-only origin st1
git checkout -b 2034_ADD_stock_picking_partner_info
git remote add custom https://github.com/qrtl/rpl-private
git fetch custom
git format-patch --keep-subject --stdout origin/st1..custom/2014_2034_imp_stock_view_adj_rpl -- stock_picking_partner_info | git am -3 --keep
git push origin HEAD
```

**Review by TomohiroAbe-Quartile — APPROVED** [review]
Code side: LGTM

File: stock_picking_partner_info/views/stock_picking_views.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_id']" position="attributes">
+                <attribute name="context">{'show_address': 1}</attribute>
+                <attribute name="options">{'always_reload': 1}</attribute>
```
Comment (yostashiro) [inline stock_picking_partner_info/views/stock_picking_views.xml:10]:
```suggestion
                <attribute name="options">{'always_reload': 1}</attribute>
                <attribute name="attrs">{'invisible': [('picking_type_code', '!=', 'outgoing')]}</attribute>
```

File: stock_picking_partner_info/views/stock_picking_views.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            </xpath>
+            <xpath expr="//field[@name='partner_id']" position="after">
+                <field name="phone" widget="phone" />
```
Comment (yostashiro) [inline stock_picking_partner_info/views/stock_picking_views.xml]:
```suggestion
                <field name="phone" widget="phone" attrs="{'invisible': [('picking_type_code', '!=', 'outgoing')]}"/>
```

File: stock_picking_partner_info/models/stock_picking.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Copyright 2021 Quartile Limited
```
Comment (yostashiro) [inline stock_picking_partner_info/models/stock_picking.py:1]:
```suggestion
# Copyright 2021 Quartile Limited
# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
```

File: stock_picking_partner_info/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1,3 @@
+This module does the following:
+
+* Display address in partner field and add phone field to the transfer order form.
```
Comment (yostashiro) [inline stock_picking_partner_info/readme/DESCRIPTION.rst]:
```suggestion
* Displays the partner address and phone in the transfer form for outgoing deliveries.
```

File: stock_picking_partner_info/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Stock Picking Partner Info",
+    "version": "12.0.1.0.0",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "category": "Stock",
+    "license": "LGPL-3",
+    "depends": [],
```
Comment (yostashiro) [inline stock_picking_partner_info/__manifest__.py]:
```suggestion
```

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Comment by tack-yabeQuartile** [comment]
temporary changed to DRAFT.
An error happened when installing/updateing this module.

**Review by yostashiro — APPROVED** [review]

**Pull request merged**